lib/restforce/db/worker.rb

```diff
@@ -125,7 +125,22 @@ def perform
             runner.load_timestamps(reader)
           end
 
-          forked.run
+          begin
+            forked.run
+          rescue => e
```

Comment:
What's the rationale for not rescuing `LoadError` specifically? Have we already run into other exceptions we need to rescue?